### PR TITLE
Ticket four

### PR DESCRIPTION
**Pull request by KarlLuther**
Hello

I am doing this pull request to basically understand what I am required to have, to be able to merge.

I have not done any styling because I personally feel like first of all I need to get all the functionality sorted. Because what's the point of making the page look pretty, if it isn't working. Also I feel like styling is something I might spend way too much time on.  Although this is not something I am supposed to do, because I am not a designer

So in fact I am doing this pull request to get answer for this questions. Apart from that, the core functionality seems to be working fine

Thank you
Ivan

### CODE REVIEW

**Review by callumu21 — CHANGES_REQUESTED** [review]
Nice start, Ivan, and I think the level of styling you've applied so far is good! Just make sure to consider mobile-first design for better responsiveness and accessibility. As for the core functionality, displaying all of the articles looks good, but you're just missing a loading state and some feedback to the user when a fetch request is still ongoing.

Once that's added, please resubmit this PR and we can take another look 👍

File: src/components/ReviewSearchPage.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { fetchItems } from "../api";
+
+const ReviewSearchPage = () => {
+  const [listItems, setListItems] = useState([]);
+  const [selectedCategory, setSelectedCategory] = useState("");
+  useEffect(() => {
+    fetchItems(selectedCategory).then((items) => {
```
[the remark this answers is not in the document]
Comment (KarlLuther) [reply src/components/ReviewSearchPage.jsx]: Understood, done😎

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const ncGamesApi = axios.create({
```
Comment (callumu21) [inline src/api.js:3]: Really nice!

File: src/components/ReviewSearchPage.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  useEffect(() => {
+    fetchItems().then((items) => {
+      setListItems(items);
+    });
+  }, [selectedCategory]);
+  if (isLoading)
+    return (
+      <section>
+        <p>
```
Comment (callumu21) [inline src/components/ReviewSearchPage.jsx:25]: I appreciate the sense of humour here. Looking forward to seeing a stylish 404 page next 👀

**Review by callumu21 — APPROVED** [review]
Great job addressing my previous comments. This should be good to merge now 👍

**Pull request merged**